themes-default/slim/test/specs/router.spec.js

```diff
@@ -0,0 +1,6 @@
+import test from 'ava';
+import router from '../../src/router';
+
+test('router compiles', t => {
+    t.truthy(router);
```

Comment:
I think [`notThrows`](https://github.com/avajs/ava#notthrowsfn-message) would work better here.

Reply:
`notThrows` requires a function, but the import happens at the top of the file.